openerp/modules/registry.py

```diff
@@ -415,7 +415,8 @@ def delete(cls, db_name):
         """Delete the registry linked to a given database.  """
         with cls.lock():
             if db_name in cls.registries:
-                cls.registries[db_name].clear_caches()
+                cls.signal_caches_change(db_name)
+                cls.signal_registry_change(db_name)
```

Comment:
Signaling changes does not make sense here, IMHO.
There should be no pending invalidation left on the registry.
I believe that the purpose of `clear_caches()` is to speed up memory deallocation...

Reply:
Based on some digging around, I think this was introduced at [a time where the cache was located outside of its registry](https://github.com/odoo/odoo/commit/b5daffc115f77d02b8e78ab6c437776f9d91e97c#diff-6a59d8ada6e3d58bf74e9fc0691b50acR157), and it made sense to clean it up upon deletion. 

This line should presumably have been dropped during the last cache refactoring. Just removing the clear_caches() call should be enough to fix #25029. Changing the registry management in WorkerCron is another topic.